packages/shared/src/consts/providers/channels/chat.ts

```diff
@@ -91,4 +92,13 @@ export const chatProviders: IProviderConfig[] = [
     docReference: 'https://developers.facebook.com/docs/whatsapp/cloud-api',
     logoFileName: { light: 'whatsapp-business.svg', dark: 'whatsapp-business.svg' },
   },
+  {
+    id: ChatProviderIdEnum.ChatWebhook,
+    displayName: 'Chat Webhook',
+    channel: ChannelTypeEnum.CHAT,
+    credentials: chatWebhookConfig,
+    docReference: `https://docs.novu.co/channels-and-providers/chat/chat-webhook${UTM_CAMPAIGN_QUERY_PARAM}`,
```

Comment:
It's not relevant to this PR, but I do not understand this campaign. Do you know, by any chance, why we use the same one for all providers `in-app`?